docs/manual/kinds/ixia-c-one.md

```diff
@@ -0,0 +1,33 @@
+# Ixia-c-one
+
+ixia-c-one is a re-packaged (as a single-container) flavor of multi-container application[ixia-c](https://github.com/open-traffic-generator/ixia-c).
```

Comment:
since we now have support for pod-like containers arrangement maybe you can test this? this will eliminate aggregating ixia-c containers into one

Reply:
Aggregation of ixia-c containers actually help in following aspects,
- keeping the topology file simple (one container vs multiple containers)
- also keeps topology simple when higher scaling of end-points (TE container) is intended
- would keep the topology agnostic of free and commercial solution (which might have more containers)

Reply:
Yes @hellt , the plan is to bring out ixia-c-one as an artifact of ixia-c release.